node/node.go

```diff
@@ -181,6 +184,10 @@ type Node struct {
 		failedTxns        *ring.Ring
 	}
 	unixTimeAtNodeStart int64
+
+	// SelfAddresses holds the addresses of bls keys run by the node
+	SelfAddresses    map[string]common.Address
```

Comment:
this naming is really awkward in golang and always was in codebase, `Self`<anything> 

lets remove and make more meaningful, maybe like ValidatorAddrs, or something like that

Reply:
sure. how about just `KeysToAddrs`? don't want to use `Validator...` as node is not concerned with validator. Any other suggestions?

Reply:
using `KeysToAddrs`